pkgs/development/libraries/libowlevelzs/default.nix

```diff
@@ -0,0 +1,40 @@
+{ cmake
+, fetchurl
+, lib
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libowlevelzs";
+  version = "0.0.3";
+
+  src = fetchurl {
+    url = "https://ytrizja.de/distfiles/${pname}-${version}.tar.gz";
+    sha256 = "49892b9ccd0943c23dc5af842f929ce9467ea50abd6a0e985f506464e8e8bfe0";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  postPatch = ''
+    patchShebangs config_relink.sh
+    substituteInPlace src/hash.hpp \
+      --replace cstdint stdint.h \
```

Comment:
substituteStream(): WARNING: pattern 'cstdint' doesn't match anything in file 'src/hash.hpp'